zcash_client_backend/src/fees.rs

```diff
@@ -328,6 +356,9 @@ pub trait ChangeStrategy {
         sapling: &impl sapling::BundleView<NoteRefT>,
         #[cfg(feature = "orchard")] orchard: &impl orchard::BundleView<NoteRefT>,
         dust_output_policy: &DustOutputPolicy,
+        #[cfg(feature = "transparent-inputs")] ignore_change_memo: bool,
+        #[cfg(feature = "transparent-inputs")] ephemeral_input_amounts: &[NonNegativeAmount],
+        #[cfg(feature = "transparent-inputs")] ephemeral_output_amounts: &[NonNegativeAmount],
```

Comment:
Why are these slices instead of `Option<NonNegativeAmount>`? Under what conditions is it necessary to create more than one ephemeral output, or consume more than one ephemeral input?

Reply:
See https://github.com/zcash/librustzcash/pull/1257/files#r1653602492